tests/Propel/Tests/Runtime/ServiceContainer/StandardServiceContainerTest.php

```diff
@@ -190,31 +192,106 @@ public function testSetAdaptersRemovesExistingAdaptersForAllDatasources($value =
     /**
      * @return void
      */
-    public function testCheckInvalidVersion()
+    public function testCheckInvalidVersion(): void
     {
-        $logger = $this->getMockBuilder('Monolog\Logger')
-            ->setMethods(['warning'])
-            ->setConstructorArgs(['mylogger'])
-            ->getMock();
-        $logger->expects($this->once())->method('warning');
+        $this->expectException(PropelException::class);
+        $this->sc->checkVersion(-1);
+    }
 
-        $this->sc->setLogger('defaultLogger', $logger);
-        $this->sc->checkVersion('1.0.0-invalid');
+    /**
+     * @doesNotPerformAssertions
+     *
+     * @return void
+     */
+    public function testCheckValidVersion(): void
+    {
+        try {
+            $this->sc->checkVersion(StandardServiceContainer::CONFIGURATION_VERSION);
+        } catch (PropelException $e) {
+            $this->fail('The current configuration version should pass a check');
+        }
     }
 
     /**
      * @return void
      */
-    public function testCheckValidVersion()
+    public function testUninitializedDatabaseMapThrowsException(): void
     {
-        $logger = $this->getMockBuilder('Monolog\Logger')
-            ->setMethods(['warning'])
-            ->setConstructorArgs(['mylogger'])
-            ->getMock();
-        $logger->expects($this->never())->method('warning');
+        $sc = new StandardServiceContainer();
```

Comment:
same, but for tests it's ok

Reply:
I agree, but `sc` is used throughout the whole class, so for the sake of consistency, I think `$sc` works best here.
If you insist, I can easily replace all occurrences. Though the PR is quite large as it is.